### DISCUSSION TITLE
Explications détaillées Sophia Script

### DISCUSSION THREAD

**Discussion by stepmond** [farag2/Sophia-Script-for-Windows#307] [Show and tell]
Bonsoir, j'ai eu le plaisir d'écrire un descriptif détaillé des fonctions de Sophia Script en Français (le plus long article que j'ai pu faire) [depuis ce lien](https://www.pcsoleil.fr/successeur-de-win10-initial-setup-script-sophia-script-comment-lutiliser/).

Il y a quelques temps j'avais fait la même chose pour feu le script Win10 Initial Setup Script. Après l'abandon par le développeur, j'ai vérifié et testé les nombreux scripts de réglage vraiment complet, j'ai choisit Sophia Script bien sur!

**Comment by farag2** [comment]
Wow! i'm not a Frech native speaker. I'd prefer you to use English. I had to use the browser extension for translation. :) Thanks for you work. I'll read your article. Of course, it will be much informative if you used the English language in you work...

**Reply by farag2** [reply]
I posted a news in my Telegram channel. This is incredible: you reviewed all functions from the preset. :smile_cat:

**Reply by stepmond** [reply]
Thanks; I thin it's better for me and more compréhensible for French people; i'll let this hard working for english native speaker ;)

**Reply by farag2** [reply]
He-he.

**Comment by farag2** [comment]
Placed you link on the main page in the table.
